tests/unit/test_configuration.py

```diff
@@ -549,20 +570,32 @@ def test_validate_choose_wrong_option_configuration(caplog: LogCaptureFixture) -
 def test_no_noise() -> None:
     # Tests that passing the sentinal no noise value results in a configuration
     # where all noise levels are 0.0
-    no_noise_config = get_configuration("no_noise").to_dict()
-
-    for dataset in no_noise_config.keys():
-        dataset_dict: dict = no_noise_config[dataset]
-        dataset_row_noise_dict: dict = dataset_dict[Keys.ROW_NOISE]
-        dataset_column_dict: dict = dataset_dict[Keys.COLUMN_NOISE]
-        for row_noise_type in dataset_row_noise_dict.keys():
-            noise_specific_config: dict = dataset_row_noise_dict[row_noise_type]
-            for key, value in noise_specific_config.items():
-                assert dataset_row_noise_dict[row_noise_type][key] == 0.0
-        for column in dataset_column_dict.keys():
-            column_noise_dict: dict = dataset_column_dict[column]
-            for column_noise_type in column_noise_dict.keys():
-                assert column_noise_dict[column_noise_type][Keys.CELL_PROBABILITY] == 0.0
+    no_noise_config = get_configuration("no_noise")
```

Comment:
I put some asserts in here and ran pytest when I was replacing dictionary references to make sure the replacements had the same values as the old variables.